### PR TITLE
release artifacts for release v0.5.2

### PR DESCRIPTION
**Pull request by ananth102**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by surajkota** [comment]
Bump version to 0.6.0 since we are adding feature

**Comment by AlmogBaku** [comment]
any update?

File: helm/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sagemaker-chart
 description: A Helm chart for the ACK service controller for Amazon SageMaker (SageMaker)
-version: v0.5.1
-appVersion: v0.5.1
+version: v0.5.2
+appVersion: v0.5.2
```
Comment (AlmogBaku) [inline helm/Chart.yaml:5]:
```suggestion
version: v0.6.0
appVersion: v0.6.0
```

File: helm/templates/NOTES.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{ .Chart.Name }} has been installed.
-This chart deploys "public.ecr.aws/aws-controllers-k8s/sagemaker-controller:v0.5.1".
+This chart deploys "public.ecr.aws/aws-controllers-k8s/sagemaker-controller:v0.5.2".
```
Comment (AlmogBaku) [inline helm/templates/NOTES.txt:2]:
```suggestion
This chart deploys "public.ecr.aws/aws-controllers-k8s/sagemaker-controller:v0.6.0".
```

**Review by AlmogBaku — CHANGES_REQUESTED** [review]

**Pull request closed without merging**